Rakefile

```diff
@@ -294,8 +294,8 @@ namespace :travis do
   end
 
   desc "Updates the travis files and creates a PR"
-  task :create_pr_with_updates do
-    force_update update_travis_files_in_repos
+  task :create_pr_with_updates, :custom_pr_comment do |t, args|
+    force_update(update_travis_files_in_repos, args[:custom_pr_comment])
```

Comment:
Whats this when not provided? `nil` or `""`, as if its `""` it should probably set it to nil.

Reply:
With the last commit (https://github.com/rspec/rspec-dev/pull/240/commits/3b1878ca70967dafb7906f5d38b1554996951431) if it's `nil` or an empty string. We do not add the two extra blank line and try to add the custom pull request comment.

I also fixed a missing comma with that task.

Initial PR updated with command result.